libs/net/dal/Configuration/BaseTypeConfiguration`.cs

```diff
@@ -17,7 +17,7 @@ public override void Configure(EntityTypeBuilder<TBaseEntity> builder)
         builder.Property(m => m.IsEnabled);
         builder.Property(m => m.SortOrder).HasDefaultValue(0);
 
-        builder.HasIndex(m => new { m.IsEnabled, m.Name }, "IX_is_enabled");
+        builder.HasIndex(m => new { m.IsEnabled, m.Name }, $"IX_{typeof(TBaseEntity).Name.ToLower()}_is_enabled");
```

Comment:
This creates unique names for each index to reduce issues when we add DB migrations.